### ISSUE TITLE
A better way to deal with exceptions?

### ISSUE THREAD

**Issue by joseims** [joseims/InsideOutDebugger#5]
**What:**
Today when an exception occur, we save it to EXCEPTION variable, and print the traceback on the screen. There's probably a better way to do that.

**Why:**
It kinda feel's weird to be treating and error in multiple way's, it should be one classy way to do it
**How:**
I don't know! Let's discuss it!

**Comment by Navneet78** [comment]
Yeah Maybe we can use "finally" block depends on situation. Or we can handle by specifying type of errors.
